cluster/gce/upgrade.sh

```diff
@@ -33,7 +33,7 @@ source "${KUBE_ROOT}/cluster/kube-util.sh"
 function usage() {
   echo "!!! EXPERIMENTAL !!!"
   echo ""
-  echo "${0} [-M|-N|-P] -l -o | <version number or publication>"
+  echo "${0} [-M|-N|-P] [-o] -l | <version number or publication>"
```

Comment:
-o only has an effect on the nodes, so it only matters if you also pass -N. Actually, since we upgrade nodes (and master) by default if you pass nothing, you could also do upgrade.sh -o <version>, so maybe you are correct...